docs/organizer-guide/confs/cfp.rst

```diff
@@ -140,3 +140,50 @@ Biases
 * New people
 * Community members
 * Previous lighting talk speakers
+
+Speaker Mentoring
+-----------------
+
+We offer to pair any speakers who'd like guidance with an experienced speaker from a previous year.
+
+Mentor selection
+~~~~~~~~~~~~~~~~
+
+Mentors have usually spoken at several conferences in the past and have a good sense of what preparation is needed.
+
+Meeting structure
+~~~~~~~~~~~~~~~~~
+
+Flexible according to the needs and availability of participants. Use the following guidelines if they are useful
+
+We met four times in the months leading up to the conference. Each meeting was an hour and there were about two to three weeks between each.
+
+Introduction/level-set:
+   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk to him.
```

Comment:
```suggestion
   Meet one another for the first time, I introduced my topic and experience with it, we decided on our meeting structure and a loose schedule. We agreed I would create an outline for my talk, then the slides, then present a practice talk to them.
```

Reply:
There's a couple other places this is gendered. Not sure if we care or not.

Reply:
Considering the new speaker / mentor dynamic we definitely care. Thanks for pointing out. <3